posthog/api/routing.py

```diff
@@ -40,11 +48,12 @@ class TeamAndOrgViewSetMixin(_GenericViewSet):
     # Example: {"team_id": "foo__team_id"} will make the viewset filtered by obj.foo.team_id instead of obj.team_id
     filter_rewrite_rules: Dict[str, str] = {}
 
-    include_in_docs = True
-
     authentication_classes = []
     permission_classes = []
 
+    # NOTE: Could we type this? Would be pretty cool as a helper
+    base_scope: Optional[APIScopeObjectOrNotSupported] = None
+    required_scopes: Optional[list[str]] = None
```

Comment:
One thing I'm slightly confused about: how does `base_scope` differ from  `required_scope`? Aren't both required?

Reply:
I'll rename it to `scope_object` or something - its the object without the action so that the actions can be derived based on the drf action performed. 

Required scopes is mostly for @action decorators as there you are specifying the specific scope requirement